manifest.md

```diff
@@ -4,124 +4,100 @@ draft = true
-in the Docker daemon in the [v1.3.0
-release](https://github.com/docker/docker/commit/9f482a66ab37ec396ac61ed0c00d59122ac07453)
-and is specified in the [schema 1 manifest definition](./manifest-v2-1.md)
+# Image Manifest
 
-This second schema version has two primary goals. The first is to allow
-multi-architecture images, through a "fat manifest" which references image
-manifests for platform-specific versions of an image. The second is to
-move the Docker engine towards content-addressable images, by supporting
-an image model where the image's configuration can be hashed to generate
-an ID for the image.
+The first goal is content-addressable images, by supporting an image model where the image's configuration can be hashed to generate an ID for the image and its components.
+The second goal is to allow multi-architecture images, through a "fat manifest" which references image manifests for platform-specific versions of an image.
+The third goal is to be translatable to the [OpenContainers/runtime-spec](https://github.com/opencontainers/runtime-spec)
 
 # Media Types
 
-The following media types are used by the manifest formats described here, and
-the resources they reference:
+The following media types are used by the manifest formats described here, and the resources they reference:
 
-- `application/vnd.docker.distribution.manifest.v1+json`: schema1 (existing manifest format)
-- `application/vnd.docker.distribution.manifest.v2+json`: New image manifest format (schemaVersion = 2)
-- `application/vnd.docker.distribution.manifest.list.v2+json`: Manifest list, aka "fat manifest"
-- `application/vnd.docker.image.rootfs.diff.tar.gzip`: "Layer", as a gzipped tar
-- `application/vnd.docker.container.image.v1+json`: Container config JSON
+- `application/vnd.oci.image.manifest.v1+json`: Image manifest format
+- `application/vnd.oci.image.manifest.list.v1+json`: Manifest list, aka "fat manifest"
+- `application/vnd.oci.image.serialization.v1+json`: Container config JSON
+- `application/vnd.oci.image.rootfs.tar.gzip`: "Layer", as a gzipped tar
 
 ## Manifest List
 
-The manifest list is the "fat manifest" which points to specific image manifests
-for one or more platforms. Its use is optional, and relatively few images will
-use one of these manifests. A client will distinguish a manifest list from an
-image manifest based on the Content-Type returned in the HTTP response.
+The manifest list is the "fat manifest" which points to specific image manifests for one or more platforms.
+Its use is optional, and relatively few images will use one of these manifests.
+A client will distinguish a manifest list from an image manifest based on the Content-Type returned in the HTTP response.
 
 ## *Manifest List* Field Descriptions
 
 - **`schemaVersion`** *int*
-	
-  This field specifies the image manifest schema version as an integer. This
-  schema uses the version `2`.
+
+  This REQUIRED property specifies the image manifest schema version.
```

Comment:
k. changed.

On Fri, Apr 8, 2016 at 2:21 PM, Stephen Day <notifications@github.com>
wrote:

> In manifest.md
> <https://github.com/opencontainers/image-spec/pull/12#discussion_r59068108>
> :
>
> >
> >  ## *Manifest List* Field Descriptions
> >
> >  - **`schemaVersion`** *int*
> > -	
> > -  This field specifies the image manifest schema version as an integer. This
> > -  schema uses the version `2`.
> > +
> > +  This REQUIRED property specifies the image manifest schema version.
>
> This reset will make the format incompatible with Docker's manifest
> format. I'd rather we maintain this at 2 and deprecate the field.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/12/files/c146488d55d702f2c7103da26ac5a20dfc5f34c1#r59068108>
>